.claude/hooks/gh-setup.sh

```
@@ -0,0 +1,50 @@
+    ;;
+esac
+
+GH_VERSION="2.67.0"
+GH_TARBALL="gh_${GH_VERSION}_linux_${GH_ARCH}.tar.gz"
+GH_URL="https://github.com/cli/cli/releases/download/v${GH_VERSION}/${GH_TARBALL}"
+INSTALL_DIR="$HOME/.local/bin"
+
+mkdir -p "$INSTALL_DIR"
+
+# Download and extract
+cd /tmp
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)